env/dev/env/ios/main.cljs

```diff
@@ -4,19 +4,24 @@
             [status-im.ios.core :as core]
             [status-im.utils.handlers :as utils.handlers]
             [figwheel.client :as figwheel :include-macros true]
+            [env.config :as conf]
+            [env.utils]
             [cljs.pprint]))
 
 (enable-console-print!)
 
+(assert (exists? core/init) "Fatal Error - Your core.cljs file doesn't define an 'init' function!!! - Perhaps there was a compilation failure?")
+(assert (exists? core/app-root) "Fatal Error - Your core.cljs file doesn't define an 'app-root' function!!! - Perhaps there was a compilation failure?")
+
 (def cnt (r/atom 0))
 (defn reloader [] @cnt [core/app-root])
 (def root-el (r/as-element [reloader]))
```

Comment:
just wondering why do we need this magic?

Reply:
this is re-natal auto-generated assertions, I think we can remove them